tests/netdev_flood_flooder/main.c

```diff
@@ -0,0 +1,212 @@
+/*
+ * Copyright (C) 2019 Freie Universität Berlin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @{
+ *
+ * @file
+ * @author  Martine Lenders <m.lenders@fu-berlin.de>
+ */
+
+#include <stddef.h>
+#include <stdint.h>
+#include <stdio.h>
+
+#ifdef MODULE_AT86RF2XX
+#include "at86rf2xx.h"
+#include "at86rf2xx_params.h"
+#endif  /* MODULE_AT86RF2XX */
+
+#include "net/netdev.h"
+#include "xtimer.h"
+
+#include "netdev_flood.h"
+
+#if     defined(MODULE_AT86RF2XX)
+#define NETDEV_ADDR_LEN             (2U)
+#define NETDEV_FLOOD_HDR_SEQ_OFFSET (2U)
+/* IEEE 802.15.4 header */
+#define NETDEV_FLOOD_HDR            { 0x71, 0x98, /* FCF */                 \
+                                      0xa1,       /* Sequence number 161 */ \
+                                      0x23, 0x00, /* PAN ID 0x23 */         \
+                                      0x0e, 0x50, /* 0x500e (start of NETDEV_FLOOD_TARGET) */ \
+                                      0x0e, 0x12, /* 0x120e (start of NETDEV_FLOOD_SOURCE) */ }
+#else
```

Comment:
Why is this at86rf2xx specific?

Reply:
Because this test is trying to point out a problem in the `at86rf2xx` implementation. For other drivers other data might be needed. See also https://github.com/RIOT-OS/RIOT/pull/11256/files#diff-f29476e871b447f52f062cd754786b75R53-R54

Reply:
But the 802.15.4 header should be the same for all 802.15.4 devices - what configuration would that be?

Reply:
The 802.15.4 header, yes. I don't remember if this data was specific to the error case though.